node/src/lib.rs

```diff
@@ -8,14 +8,19 @@
     unsafe_code,
     unstable_features,
     unused_import_braces,
-    unused_qualifications,
+    unused_qualifications
```

Comment:
why not?

Reply:
why not the comma? :-)

Reply:
I actually don't know! The auto formatter removes it for me ..

Reply:
yes the comma - i expected trailing commas for multi-line enumerations. is this something we need to configure for rust specifically?

Reply:
Ja, in this case its compiler flags, telling `rustc` to change its internal linters, maybe there its different?